server/app.py

```
@@ -0,0 +1,297 @@
+        if filename.endswith(".json"):
+            file_id = filename.replace(".json", "")
+            path = os.path.join(DATA_DIR, filename)
+            try:
+                # We could load the tree to get the root content as a title, but that's expensive
+                # checking mtime for sorting
+                mtime = os.path.getmtime(path)
+                conversations.append({
+                    "id": file_id,
+                    "name": file_id, # Could improve naming later
+                    "updated_at": mtime,
+                    "id": file_id,
```

Ruff (0.14.10)

Dictionary key literal `"id"` repeated

Remove repeated key literal `"id"`

(F601)